core/src/main/java/org/jdbi/v3/core/statement/ParsedParameters.java

```diff
@@ -78,4 +79,35 @@ public String toString() {
             + ", parameterNames=" + parameterNames
             + '}';
     }
+
+    /**
+     * A static factory of named {@link ParsedParameters} instances.
+     * List given in argument must contain parameter names from SQL.
```

Comment:
*List must contain parameter names from the related statement. They must be bare names, free of SQL variable processing syntax such as a prefixed colon or other delimiters.

Reply:
Done.